lib/src/main/java/com/diffplug/spotless/sql/dbeaver/SQLFormatterConfiguration.java

```diff
@@ -18,27 +18,35 @@
 import java.util.Properties;
 
 /**
- * SQLFormatterConfiguration
+ * **Warning:** Use this class at your own risk. It is an implementation detail and is not
+ * guaranteed to exist in future versions.
+ *
+ * Forked from
+ * DBeaver - Universal Database Manager
+ * Copyright (C) 2010-2017 Serge Rider (serge@jkiss.org)
+ *
+ * Based on SQLFormatterConfiguration from https://github.com/serge-rider/dbeaver,
+ * which itself is licensed under the Apache 2.0 license.
  */
 public class SQLFormatterConfiguration {
```

Comment:
you are right, its linked to the implentation of the dbeaver formatter, It could be renamed to `DBeaverSQLFormatterConfiguration`, what do you think?

Reply:
I don't see why not, so I'll do precisely that. :)